configs/config.json.example

```diff
@@ -7,23 +7,35 @@
     "libencrypt_location": "",
     "tasks": [
       {
-        "type": "HandleSoftBan"
+        "type": "HandleSoftBan",
+        "config": {
```

Comment:
I added the NicknamePokemon task with enabled:false and removed the config from other tasks